### DISCUSSION TITLE
How to use LvGL Graphics Library | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1032] [Q&A]
# How to use LvGL Graphics Library | Seeed Studio Wiki

In this tutorial, we will teach you how to use the LvGL (Light and Versatile

https://wiki.seeedstudio.com/Wio-Terminal-LCD-Graphics/

**Comment by zol123** [comment]
I would like to remind you that the basic TFT_eSPI.h library has advanced graphics  functions such as:
   // Read the colour of a pixel at x,y and return value in 565 format
    uint16_t readPixel(int32_t x0, int32_t y0);

    // The next functions can be used as a pair to copy screen blocks (or horizontal/vertical lines) to another location
    // Read a block of pixels to a data buffer, buffer is 16 bit and the array size must be at least w * h   
    void readRect(int32_t x0, int32_t y0, int32_t w, int32_t h, uint16_t *data);
    // Write a block of pixels to the screen
    void pushRect(int32_t x0, int32_t y0, int32_t w, int32_t h, uint16_t *data);

pair readRect /  pushRect  can be used  for  graphics scrolling.....
and more..
